openbb_terminal/etf/etf_controller.py

```diff
@@ -116,21 +116,8 @@ def __init__(self, queue: List[str] = None):
                 "--limit": one_to_fifty,
                 "-l": "--limit",
             }
-            choices["holdings"] = {
```

Comment:
Tried `etf/holdings` and now it show no autocompletion at all.

Reply:
There aren't any autocomplete options for holdings